app/Http/Middleware/VerifyCsrfToken.php

```diff
@@ -28,13 +28,20 @@
 class VerifyCsrfToken extends BaseVerifier
 {
     protected $except = [
+        'api/*',
         'oauth/authorize',
         'oauth/access_token',
     ];
 
     public function handle($request, Closure $next)
     {
         try {
+            foreach ($this->except as $except) {
```

Comment:
already done in `parent::handle`?

Reply:
the parent will skip the csrf check, but still sets the csrf token cookie...
![](https://puu.sh/B6N2I/462dc8168a.png)
https://github.com/laravel/framework/blob/ab62f037fe7d2b611722801f73f2541a1c7553ec/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php#L67